test/e2e_node/container_metrics_test.go

```diff
@@ -0,0 +1,154 @@
+				"container_memory_failcnt":                      boundedSample(0, 1000000),
+				"container_memory_failures_total":               boundedSample(0, 1000000),
+				"container_memory_mapped_file":                  boundedSample(0, 1000000),
+				"container_memory_max_usage_bytes":              boundedSample(0, 80*e2evolume.Mb),
+				"container_memory_rss":                          boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
+				"container_memory_swap":                         boundedSample(0, 1000000),
+				"container_memory_usage_bytes":                  boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
+				"container_memory_working_set_bytes":            boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
+				"container_oom_events_total":                    preciseSample(0),
+				"container_processes":                           boundedSample(0, 100),
+				"container_sockets":                             boundedSample(0, 100),
+				"container_spec_cpu_period":                     preciseSample(100000),
+				"container_spec_cpu_shares":                     preciseSample(2),
+				"container_spec_memory_limit_bytes":             preciseSample(79998976),
+				"container_spec_memory_reservation_limit_bytes": preciseSample(0),
+				"container_spec_memory_swap_limit_bytes":        boundedSampleIgnoreTimestamp(0, 79998976),
+				"container_start_time_seconds":                  boundedSampleIgnoreTimestamp(time.Now().Add(-maxStatsAge).Unix(), time.Now().Add(2*time.Minute).Unix()),
+				"container_tasks_state":                         preciseSample(0),
+				"container_threads":                             preciseSample(2),
+				"container_threads_max":                         boundedSample(0, 100000),
+				"container_ulimits_soft":                        boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
+			}
+			appendMatchesForContainer(f.Namespace.Name, pod0, pod1, "busybox-container", ctrMatches, keys, gstruct.AllowDuplicates|gstruct.IgnoreExtras)
+
+			ctrOptionalMatches := map[string]types.GomegaMatcher{
+				"container_fs_io_current":            preciseSample(0),
+				"container_fs_io_time_seconds_total": preciseSample(0),
+				"container_fs_inodes_free":           boundedSample(0, 10*e2evolume.Kb),
+				"container_fs_inodes_total":          boundedSample(0, 100),
+				"container_fs_limit_bytes":           boundedSample(100*e2evolume.Mb, 10*e2evolume.Tb),
+				"container_fs_usage_bytes":           boundedSample(0, 1000000),
+				"container_fs_reads_merged_total":    preciseSample(0),
+				"container_fs_read_seconds_total":    preciseSample(0),
+				"container_fs_sector_reads_total":    preciseSample(0),
+				"container_fs_sector_writes_total":   preciseSample(0),
+				"container_fs_writes_merged_total":   preciseSample(0),
+				"container_fs_write_seconds_total":   preciseSample(0),
+			}
+			// TODO: determine why these are missing from containerd but not CRI-O
```

Comment:
@bobbypage @mikebrow @SergeyKanzhelev do y'all know why this would happen? the containerd tests here don't have these values, nor some of the network ones below